README.md

```diff
@@ -130,21 +126,14 @@ py -3.12 -m pip install plasmapy
 The `3.12` may be replaced by any version of Python that is supported by
 PlasmaPy.
 
-If you have [installed Conda], then you can also install PlasmaPy into
-an activated Conda environment by running:
+If you have [installed Conda], then you can install PlasmaPy into  an
+activated Conda environment by running:
 ```Shell
 conda install -c conda-forge plasmapy
 ```
-PlasmaPy can also be installed using [Anaconda Navigator] so long as
-`conda-forge` is added as a channel.
```

Comment:
The licensing for Anaconda Navigator has gradually been becoming more restrictive, so I'm leaning against including it in `README.md` (though it should be discussed in the online documentation).